protostar-rust/tests/data/declare_test/Scarb.toml

```
@@ -0,0 +1,10 @@
+[package]
+name = "declare_test"
+version = "0.1.0"
+
+[[target.starknet-contract]]
+sierra = true
+
+[dependencies]
+starknet = ">=1.0.0"
```

```suggestion
starknet = ">=1.1.0"
```
It should match our current compiler version

---

[fdbck](https://github.com/software-mansion/protostar/pull/2059/commits/f83f77d95fc28d6557dbd04d98a83d3bce5938b9)